service/app/controllers/scheduler/SchedulerController.java

```diff
@@ -0,0 +1,21 @@
+package controllers.scheduler;
+
+import controllers.BaseController;
+import org.sunbird.common.models.util.ActorOperations;
+import play.mvc.Http;
+import play.mvc.Result;
+
+import java.util.concurrent.CompletionStage;
+
+public class SchedulerController extends BaseController {
+ public CompletionStage<Result> startScheduler(Http.Request httpRequest) {
+  return handleRequest(
+          ActorOperations.START_SCHEDULAR.getValue(),
+          httpRequest.body().asJson(),
```

Comment:
Add job name validation as well, if user is passing three job name , and among three one job name is incorrect then provide him proper error and no job need to be run.
job should run if all provided details are correct.

Reply:
It contains only one field, this particular validation is done in actor.